app/routes/task_routes.py

```diff
@@ -1 +1,77 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, abort, make_response, Response
+from app.models.task import Task
+from datetime import datetime
+import os
+import requests
+from ..db import db
+from .helper_methods import validate_model, helper_model_from_dict, helper_get_sorted_query
+
+bp = Blueprint("tasks_bp", __name__, url_prefix = "/tasks")
+
+@bp.post("")
+def create_task():
+    request_body = request.get_json()
+
+    return helper_model_from_dict(Task, request_body)
+    
+@bp.get("")
+def get_all_tasks():
+    sort_param = request.args.get("sort") 
+    query = helper_get_sorted_query(Task, sort_param)
+    tasks = db.session.execute(query).scalars()
+    task_list = [task.to_dict() for task in tasks]
+    return task_list
+
+@bp.get("/<task_id>")
+def get_one_task(task_id):
+    task = validate_model(Task, task_id)
+    
+    return {"task": task.to_dict()}
+
+@bp.put("/<task_id>")
+def update_one_task(task_id):
+    task = validate_model(Task, task_id)
+    request_body = request.get_json()
+
+    task.title = request_body["title"]
+    task.description = request_body["description"]
```

Comment:
If the client sends a request body without the required fields 'title' or 'description' then either of these lines of code will throw an unhandled exception.

How could you add error handling so that this route returns a response indicating that the client sent a bad request in the scenario described above?